test/extended/networking/egress_firewall.go

```diff
@@ -128,6 +149,21 @@ func sendEgressFwTraffic(f *e2e.Framework, oc *exutil.CLI, pod string, nodeSelec
 	_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://docs.openshift.com").Output()
 	expectNoError(err)
 
+	// TODO:
+	// - Check for coredns-ocp-dnsnameresolver plugin.
+	// - Remove TechPreview check when feature is GA.
+	if networkPluginName() == OVNKubernetesPluginName && exutil.IsTechPreviewNoUpgrade(oc) {
+		// Test curl to `www.google.com` and `translate.google.com` should pass
+		// because we have allow dns rule for `*.google.com`.
+		g.By("sending traffic to `www.google.com` that matches allow dns rule for `*.google.com`")
+		_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://www.google.com").Output()
+		expectNoError(err)
+
+		g.By("sending traffic to `translate.google.com` that matches allow dns rule for `*.google.com`")
+		_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://translate.google.com").Output()
+		expectNoError(err)
+	}
+
```

Comment:
I think we need to update endpoint on line 170, as `http://www.google.com:` now matches the wildcard rule, and the test will fail. We should pick another dns name that is not allowed by all dns configs we have, maybe try `redhat.com`

Reply:
suggestion incorporated.

Reply:
I didn't mean to add a new check, but to change existing one here https://github.com/openshift/origin/blob/master/test/extended/networking/egress_firewall.go#L133-L134, it is now on line 172.

Reply:
Sorry, misread the comment. Code updated now.